scripts/provision-all

```diff
@@ -7,7 +7,7 @@ set -o pipefail
 
 # Format list of all VMs comma-separated, for use as qubesctl target
 all_sdw_vms_target="$(./scripts/list-vms | perl -npE 's/\n/,/g' | perl -npE 's/,$//' )"
-
+templates_to_remove="$(./scripts/list-vms old-templates | perl -npE 's/\n/ /g')"
```

Comment:
N.B. that #351 _removes_ the `list-vms` script, in favor of the new `qvm-ls --tags` support. As above, if we perform the removal in Salt, we can execute in parallel against all of them, as long as we wait until the bash script has finished.